dev-workflow-ui/webContent/resources/css/workflow-ui.css

```diff
@@ -248,16 +248,12 @@ body .custom-accordion.ui-accordion .ui-accordion-header:first-child {
 
 /* Notifications */
 .notification-badge {
-  position:absolute;
-  bottom:30px;
-  font-weight:bold;
-  left:20px;
-  font-size:13px;
-  width:22px;
-  text-align:center;
-  padding:2px;
-  height:20px;
-  color:white;
+  width:22px; height:20px;
+  font-weight:bold; font-size:13px; color:white;
+  position:absolute; bottom:30px; left:20px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
```

Comment:
i meant next to each other vertically not horizontally 😅 
 e.g:
 ```css
width:22px;
height:20px;
font-weight:bold; 
font-size:13px;
```
instead of a mix like
```css
font-size:13px;
width:22px;
font-weight:bold; 
padding:2px;
height:20px;
  ```

Reply:
![image](https://github.com/axonivy/dev-workflow-ui/assets/7498380/0dcb336c-812a-405e-891b-3c2096378b1b)

Reply:
yes i know. i should have explained it better. i was just giving an example which properties belong together